RELEASE/scripts/autoscend/paths/dark_gyffte.ash

```diff
@@ -363,16 +363,38 @@ boolean bat_shouldPickSkills(int hpLeft)
 	return false;
 }
 
+boolean bat_haveEnsorcelee() // checks if you have a current Ensorceled Monster
+{
+	if(my_class() != $class[Vampyre] || !auto_have_skill($skill[Ensorcel]))
+		return false;
+
+	if(get_property("ensorcelee") == $monster[none])
+		return false;
+	
+	return true;
+}
+
+phylum bat_ensorceledMonster() //returns phylum of current Ensorceled Monster (if you have one)
+{
+	if(!bat_haveEnsorcelee()){
+		return $phylum[none];
+	}
+
+	return monster_phylum(to_monster(get_property("ensorcelee")));
```

Comment:
```suggestion
	return monster_phylum(to_monster(get_property("ensorcelee")));
```
`to_monster("") == $phylum[none]` as such the first if is redundant

Reply:
Suggested adopted. Thanks :)